### ISSUE TITLE
WISH: in Red/System, with keyword should allow nested context abbreviation

### ISSUE THREAD

**Issue by meijeru** [red/red#3302]
Currently, the `with` keyword may be followed by a list of context names, but these are "in parallel" rather than nesting. The use case for that seems doubtful. To achieve the effect of nesting, one has to nest the `with` statements.

**Comment by greggirwin** [comment]
There is an old PR for `with` in Red: https://github.com/red/red/pull/1156

@meijeru, would you please post examples of each behavior, so we can more easily compare, and align a possible Red version with the R/S behavior?

**Comment by PeterWAWood** [comment]
Following the introduction of the [Red REP repository for enhancements and wishes](https://gitter.im/red/red?at=5adfeed527c509a7744a3dc8), this issue is being closed.

If you believe that your wish is still valid, please open a new issue for it in the new [red/REP repo](https://github.com/red/REP). It would help if you could include a link to this issue when opening an issue in the REP repository.
